src/Workspaces/Core/Portable/NamingStyles/NamingStyle.cs

```diff
@@ -163,34 +163,32 @@ private WordSpanEnumerable GetWordSpans(string name, TextSpan nameSpan)
         private static string Substring(string name, TextSpan wordSpan)
             => name.Substring(wordSpan.Start, wordSpan.Length);
 
-        private static Func<string, TextSpan, bool> s_firstCharIsLowerCase = (val, span) => !DoesCharacterHaveCasing(val[span.Start]) || char.IsLower(val[span.Start]);
-        private static Func<string, TextSpan, bool> s_firstCharIsUpperCase = (val, span) => !DoesCharacterHaveCasing(val[span.Start]) || char.IsUpper(val[span.Start]);
-
-        private static Func<string, TextSpan, bool> s_wordIsAllUpperCase = (val, span) =>
-        {
-            for (int i = span.Start, n = span.End; i < n; i++)
-            {
-                if (DoesCharacterHaveCasing(val[i]) && !char.IsUpper(val[i]))
-                {
-                    return false;
-                }
-            }
-
-            return true;
-        };
-
-        private static Func<string, TextSpan, bool> s_wordIsAllLowerCase = (val, span) =>
-        {
-            for (int i = span.Start, n = span.End; i < n; i++)
-            {
-                if (DoesCharacterHaveCasing(val[i]) && !char.IsLower(val[i]))
-                {
-                    return false;
-                }
-            }
+        private static readonly Func<string, TextSpan, bool> s_firstCharIsLowerCase = (val, span) => !DoesCharacterHaveCasing(val[span.Start]) || char.IsLower(val[span.Start]);
+        private static readonly Func<string, TextSpan, bool> s_firstCharIsUpperCase = (val, span) => !DoesCharacterHaveCasing(val[span.Start]) || char.IsUpper(val[span.Start]);
+        private static readonly Func<string, TextSpan, bool> s_wordIsAllUpperCase = (val, span) =>
+{
```

Comment:
Formatting got mangled here.

Reply:
It did. Annoyingly, it happens in the IDE but so far doesn't happen in the unit tests. Investigating...

Reply:
➡️ Formatting issues are now resolved, including these examples as well as a few others where whitespace changed unnecessarily.